root/layout/index.js

```diff
@@ -88,6 +89,28 @@ const Layout = ({$c, ...props}) => (
           </p>
         </div>}
 
+      {$c.user.birth_date &&
```

Comment:
The test failures aren't exactly unexpected: this throws an exception because `$c.user` can be undefined.